Gems/Atom/RHI/Code/Source/RHI/AliasedHeap.cpp

```diff
@@ -245,7 +246,8 @@ namespace AZ::RHI
         if (!CheckBitsAny(m_compileFlags, TransientAttachmentPoolCompileFlags::DontAllocateResources))
         {
             AZ_Assert(m_totalAllocations < m_cache.GetCapacity(),
-                "Exceeded the size of the cache! This will destroy actively used resources");
+                "Exceeded the size of the cache! This will destroy actively used resources. (Total Allocations was %i, and Cache capacity was %i)",
+                m_totalAllocations, m_cache.GetCapacity());
             m_totalAllocations++;
 
```

Comment:
SAme comment as above, it might compile on some platforms, and die on others.

Reply:
When I tested this it didnt throw any errors and worked fine, but if this wont compile on some other platforms potentially then thanks for catching it. I'll add your recommendations and ensure things still print out correctly.